src/Features/CSharp/Portable/CodeRefactorings/InlineTemporary/InlineTemporaryCodeRefactoringProvider.cs

```diff
@@ -162,7 +162,7 @@ private static bool HasConflict(IdentifierNameSyntax identifier, VariableDeclara
 
         private static SyntaxAnnotation CreateConflictAnnotation()
         {
-            return ConflictAnnotation.Create(CSharpFeaturesResources.ConflictsDetected);
+            return ConflictAnnotation.Create(CSharpFeaturesResources.Conflict_s_detected);
```

Comment:
I see, thanks. I'm good with as it is, just wanted to understand the pattern for writing it by hand.. underscores in place of parens or braces sounds easy to remember.